docs/topics/encryption/base64.md

```diff
@@ -30,4 +30,19 @@ def base64_decode(string: str) -> str:
     return base64.urlsafe_b64decode(string.encode()).decode()
 ```
 
+### **Javascript**
+
+```js
+const base64Encode = string => btoa(string).replace("+", "-").replace("/", "_");
+
```

Comment:
Why are you using callbacks, these are plain functions

Reply:
I thought it was clearer to just use arrow functions in this case because it's a simple one line thing